panel/src/components/Layout/Frame/ImageFrame.vue

```diff
@@ -43,14 +45,59 @@ export const props = {
 };
 
 /**
- * Use <k-image-frame> to display an image in a fixed ratio with background etc.
+ * Use <k-image-frame> to display an image from an external URL
+ * or internal file UUID in a fixed ratio with background etc.
  * @since 4.0.0
  *
  * @example <k-image-frame src="https://getkirby.com/image.jpg" ratio="16/9" back="pattern" />
  */
 export default {
 	mixins: [props],
-	inheritAttrs: false
+	inheritAttrs: false,
+	data() {
+		return {
+			resolvedAlt: null,
+			resolvedSrc: null,
+			resolvedSrcset: null
+		};
+	},
+	computed: {
+		isInternal() {
+			return this.src.startsWith("file://") === true;
+		}
+	},
+	watch: {
+		src: {
+			handler: "fetch",
+			immediate: true
+		}
+	},
+	methods: {
+		async fetch() {
+			if (this.isInternal === false) {
+				this.resolvedAlt = this.alt;
+				this.resolvedSrc = this.src;
+				this.resolvedSrcset = this.srcset;
+				return;
+			}
+
+			// if internal file, load data for file UUID from request endpoint
+			const data = await await this.$panel.get("items/files", {
+				query: {
+					items: this.src,
+					layout: "cards",
```

Comment:
@bastianallgeier Should we add another "type" in `Panel\Model::imageSrcset()` for this case with more/different sizes?

Reply:
Stupid question. Do we still need the different srcsets for different layouts at all anymore when the image frame sets the width? Isn't just one enough?